config/src/test/java/org/springframework/security/config/annotation/method/configuration/PrePostReactiveMethodSecurityConfigurationTests.java

```diff
@@ -212,4 +227,29 @@ ReactiveMethodSecurityService methodSecurityService() {
 
 	}
 
+	@Configuration
+	static class PermissionEvaluatorConfig {
+
+		@Bean
+		@Role(BeanDefinition.ROLE_INFRASTRUCTURE)
+		static DefaultMethodSecurityExpressionHandler methodSecurityExpressionHandler() {
+			DefaultMethodSecurityExpressionHandler handler = new DefaultMethodSecurityExpressionHandler();
+			handler.setPermissionEvaluator(new PermissionEvaluator() {
```

Comment:
Will you please use a mock permission evaluator here and then verify its usage in the test? This will provide a stronger assurance that the permission evaluator wired in the test is in use.

Please see `AuthorizationEventPublisherConfig` used by `PrePostMethodSecurityConfigurationTests#preAuthorizeWhenAuthorizationEventPublisherThenUses` for a technique that I prefer that achieves this.